Tests/Resources/Logger/TestLogger.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace OAT\Bundle\Lti1p3Bundle\Tests\Resources\Logger;
+
+use Psr\Log\AbstractLogger;
```

Comment:
Why didn't you simply require `psr/log`?
This bundle clearly depends on it, regardless of the tests.

Reply:
The library still depends on `psr/log`.
1. Please add the dependency on `psr/log` `^1.0.1 || ^2.0 || ^3.0` into the production requirements
2. Please add the dependency on `psr/log` `^1.0.1` into the dev requirements

This will solve the tests problem _and_ will explicitly require `psr/log` dependency.

Reply:
Good